app/frontend/selectors.ts

```diff
@@ -0,0 +1,18 @@
+import {State} from './state'
+import {BIG_DELEGATOR_THRESHOLD, PREMIUM_MEMBER_BALANCE_TRESHOLD} from './wallet/constants'
+
+export const totalWalletBalanceSelector = (state: State) =>
+  state.accountsInfo.reduce((a, {balance}) => balance + a, 0)
+
+export const totalRewardsBalanceSelector = (state: State) =>
+  state.accountsInfo.reduce((a, {shelleyBalances}) => shelleyBalances.rewardsAccountBalance + a, 0)
+
+export const isBigDelegatorSelector = (state: State) => {
+  const totalWalletBalance = totalWalletBalanceSelector(state)
+  return totalWalletBalance > BIG_DELEGATOR_THRESHOLD
```

Comment:
now, the big question is where should business logic like this reside :thinking: I am not a big fan of hard-coding it in the selectors but I don't see a better place for it right now

Reply:
I think that e.g. this could be exposed by `wallet` module, not exposing the `BIG_DELEGATOR_THRESHOLD` constant (it could be hidden to the outside world), but exposing something like "isBigDelegator" function. Though I believe such refactor is out the scope of this PR, but I believe there will be time for such changes in the near future :)